frontend/src/components/IstioWizards/GatewaySelector.tsx

```diff
@@ -217,20 +243,29 @@ export class GatewaySelector extends React.Component<Props, GatewaySelectorState
             </FormGroup>
             {!this.state.newGateway && (
               <FormGroup fieldId="selectGateway" label={t('Gateway')}>
-                {this.props.gateways.length > 0 && (
-                  <FormSelect
-                    id="selectGateway"
-                    value={this.state.selectedGateway}
-                    isDisabled={!this.state.addGateway || this.state.newGateway || this.props.gateways.length === 0}
-                    onChange={(_event, gw: string) => this.onFormChange(GatewayForm.GATEWAY_SELECTED, gw)}
-                  >
+              {this.props.gateways.length > 0 && (
+                <Select
+                  id="selectGateway"
+                  isOpen={this.state.isOpen}
+                  selected={this.state.selectedGateway}
+                  onSelect={this.onSelect}
```

Comment:
you need to call `this.onFormChange` function to maintain the same application behaviour. In this case, you will have to add `isOpen: false` to the `this.onFormChange` function:

```
case GatewayForm.GATEWAY_SELECTED:
        this.setState(
          {
            selectedGateway: value,
            isOpen: false
          },
          () => this.props.onGatewayChange(this.isGatewayValid(), this.state)
        );
        break;
```

```suggestion
                  onSelect={(_event, gw) => this.onFormChange(GatewayForm.GATEWAY_SELECTED, gw as string)}
```

Reply:
resolved